### ISSUE TITLE
npm failing to install @design-systems

### ISSUE THREAD

**Issue by atrakh** [intuit/design-systems-cli#624]
**Describe the bug**

Hello!

I'm following the [instructions for installing the cli](https://intuit.github.io/design-systems-cli/#/getting-started), and am receiving an npm error when initializing `package.json`

**To Reproduce**

From an empty directory, run `npm init @design-systems`

```
> npm --version
7.5.6
> npm init @design-systems
Need to install the following packages:
  @design-systems/create
Ok to proceed? (y) y
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:../cli-utils
```

Some research suggests that using `file` instead of `link` for local dependencies may resolve this issue. (ref: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#dependencies)

**Comment by tylerkrupicka** [comment]
@kendallgassner has a handy script for finding the issue, we may need to set that up!

**Comment by kendallgassner** [comment]
I do have a local-dependency rule.  But this looks unrelated to casing :/

https://www.npmjs.com/package/@kendallgassner/eslint-plugin-package-json

**Comment by adierkens** [comment]
You can't use `link:` in a peerDependency: https://github.com/intuit/design-systems-cli/blob/master/packages/create/package.json#L29

It doesn't get updated by yarn during the release process

**Comment by hipstersmoothie** [comment]
@kendallgassner we should just change those links to ranges:

```json
{
  "peerDependencies": {
     "@design-systems/cli-utils": ">= 2.x",
     "@design-systems/plugin": ">= 2.x",
  }
}
```

**Comment by kendallgassner** [comment]
^ ill add this to my rule

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v2.12.3`](https://github.com/intuit/design-systems-cli/releases/tag/v2.12.3) :rocket:
